takwimu/settings.py

```diff
@@ -40,6 +40,23 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'takwimu',
+    #wagtail
+    'wagtail.contrib.forms',
+    'wagtail.contrib.redirects',
+    'wagtail.embeds',
+    'wagtail.sites',
+    'wagtail.users',
+    'wagtail.snippets',
+    'wagtail.documents',
+    'wagtail.images',
+    'wagtail.search',
+    'wagtail.admin',
+    'wagtail.core',
```

Comment:
Wouldn't the wagtail components be installed in the `blog` app instead?

Reply:
@DavidLemayian do we plan on deploying the blog app separate from the main `takwimu` app, kinda like HURUmap? Because installing the wagtail components in the `blog` app would mean having a separate `settings.py` for `blog`

Reply:
@phillipahereza It's not for deployment purposes but more development. Separation of concerns + re-useable apps - https://docs.djangoproject.com/en/2.0/intro/reusable-apps/